src/Service/PdfCreator.php

```diff
@@ -4,38 +4,36 @@
 
 namespace App\Service;
 
-use Spipu\Html2Pdf\Html2Pdf;
+use Dompdf\Dompdf;
+use Dompdf\Options;
 
 final class PdfCreator implements PdfCreatorInterface
 {
     private $pdf;
     private $orientation;
     private $format;
-    private $lang;
-    private $unicode;
-    private $encoding;
-    private $margin;
 
-    public function __construct(string $orientation, string $format, string $lang, bool $unicode, string $encoding, array $margin)
+    public function __construct(string $orientation, string $format)
     {
         $this->orientation = $orientation;
         $this->format = $format;
-        $this->lang = $lang;
-        $this->unicode = $unicode;
-        $this->encoding = $encoding;
-        $this->margin = $margin;
     }
 
     public function create(): void
```

Comment:
Can you add $isHtml5ParserEnabled and $isRemoteEnabled in parameters of the function with true value by default? to cover changes in creation, and also a third parameter with object Dompdf/Options null by default and in the function `If (null !== $options) { new Dompdf($options) }`
Thanks :)